src/components/list/list.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState} from 'react'
+
+import { Whapper, WhapperTitle, UsersName, UsersEmail, Container, BoxName, BoxEmail, Navigation, BtnPrevious, BtnNext } from './list-style'
+import { getUsersQuery } from "../../service/get-user-query";
+import { useQuery } from '@apollo/client';
+import { useNavigate } from 'react-router-dom';
+
+
+interface UserType {
+  email: 'string',
+  name: 'string',
+  id: number,  
+}
+export const ListUsers = ():JSX.Element => {
+  const navigate = useNavigate();
+  const token = localStorage.token;
+  const limit = 20;
+  const [offset, setOfsset] = useState(0);
+  const { data } = useQuery(getUsersQuery, {
+  context: {
+      headers: {
+        Authorization: token,
+      },
+    },
+    variables: {
+      data: {
+        offset: offset,
+        limit: limit
+      }
+      }
+    ,  
+  });
+  console.log(data);
+ 
+  const nextPageexists = data?.users?.pageInfo?.hasNextPage;
+  const previousPageexists = data?.users?.pageInfo?.hasPreviousPage;
+
+  const nextPage = () => {
+    setOfsset(offset + limit);
+  };
+  const previusPage = () => {
+    setOfsset(offset - limit);
+  };
+
+    return (
+    <Whapper>
+      <WhapperTitle> Lista de Usuarios</WhapperTitle>
+      <Container>
+        <BoxName>
+          <UsersName>Nome</UsersName>
+          {data?.users?.nodes?.map((user:UserType)=>
+          <p key={user.id}>{user.name}</p>)}
```

Comment:
Aqui, vale aquela questão que conversamos, vai dar problema pq teremos duas keys iguais aqui no nome e no map abaixo do email, por isso seria legal usar uma template string para diferencia-los...
```suggestion
          <p key={`name-column-${user.id}`}>{user.name}</p>)}
```

e replicar o mesmo na debaixo para manter a consistência